README.md

```diff
@@ -19,14 +19,39 @@ The main features include:
 
 Read more about it on the [OpenRocket Wiki](http://wiki.openrocket.info).
 
+Running OpenRocket
```

Comment:
This is a great  section to add.   

However, the first thing mentioned should be the installers: (which we don't yet have a download page on the website... but we could still mention them, and add the url later?)

I wouldn't even recommend running the jars *directly*, like the below.   If they're going to github, wouldn't they want to use an IDE?   Like eclipse, Intellij or VSCode? 
Should we get people to contribute instructions for each of those?

Reply:
Please don't get rid of instructions for building and running directly from the command line (in the wiki instead of the README is fine, just don't get rid of them completely). This may reflect the fact that I'm older than dirt, but every time I've tried to use an IDE I've felt like I was trying to thread a needle while wearing boxing gloves. I'm doing all my editing in emacs and using good ol' print statements for debugging.

Reply:
@JoePfeiffer wiki updated: https://github.com/openrocket/openrocket/wiki/Instructions-to-Build-from-Terminal

@teyrana Removed Running OpenRocket section to be deferred to the wiki above. Added Installers section.

Reply:
I just made a slight change to the instructions -- corrected where to build from, and said where the JAR ends up.

Reply:
Okay, that's fine, I have no object to leaving the native builds in there. . We should probably mention all the methods, here.  

The more I think about it, this README is probably an entry point for new developers -- so it should mention enough to get people started, for their preferred method. Not all the details-- it sounds like y'all want to move full details to the wiki.   But enough 

I do prefer repository readme  files to stand on their own.... but they don't need to answer all the questions.

Reply:
> @teyrana What if we remove the file and use the wiki instead? https://github.com/openrocket/openrocket/wiki/Change-Log

Actually, now that I think about this more. The Changelog should not be a wiki entry. It can be a page on openrocket.github.io but it should *not* be a wiki entry which is editable.